jib-cli/src/main/java/com/google/cloud/tools/jib/cli/war/WarFiles.java

```diff
@@ -77,12 +77,24 @@ private static List<String> computeEntrypoint(
       CommonContainerConfigCliOptions commonContainerConfigCliOptions)
       throws InvalidImageReferenceException {
     List<String> entrypoint = commonContainerConfigCliOptions.getEntrypoint();
+    String baseImage = fetchBaseImage(commonContainerConfigCliOptions);
     if (!entrypoint.isEmpty()) {
       return entrypoint;
     }
     if (commonContainerConfigCliOptions.isJettyBaseimage()) {
```

Comment:
What if the `baseImage` is jetty but due to the default rather than being specified by CLI options?

Reply:
The isJettyBaseImage() method contains some logic to check if the user-specified base image is jetty. If the base  image is not customized then we assume that the default (`jetty:latest`) will be used and return `true`. Reference: https://github.com/GoogleContainerTools/jib/blob/6df635ee51bc466c961cdeea23d84e84e87ba4b0/jib-cli/src/main/java/com/google/cloud/tools/jib/cli/CommonContainerConfigCliOptions.java#L189-L197